tests/test_tier_lookup.py

```diff
@@ -0,0 +1,128 @@
+from datetime import datetime
+from freezegun import freeze_time
+from unittest.mock import Mock
+from freezegun.api import FakeDate
+
+from vulnerable_people_form.tier_lookup.lacode_tier_loookup import (
+    LacodeTierLookup,
+    PATH_TO_RESTRICTIONS_FILE,
+    DEFAULT_NATIONAL_ALERT_LEVEL
+)
+
+
+def test_lacode_to_tier_single_restriction():
+    lacode = "E08000011"
+    test_object = LacodeTierLookup(tier_file_path=PATH_TO_RESTRICTIONS_FILE)
+    tier = test_object.lacode_to_tier(lacode=lacode)
+    assert tier == 3
+
+
+def test_lacode_to_tier_multiple_restriction():
+    lacode = "E08000001"
+    test_object = LacodeTierLookup(tier_file_path=PATH_TO_RESTRICTIONS_FILE)
+    tier = test_object.lacode_to_tier(lacode=lacode)
+    assert tier == 3
+
+
+@freeze_time('2020-10-20')
+def test_lacode_to_tier_multiple_restriction_previous_date():
+    lacode = "E08000018"
+    test_object = LacodeTierLookup(tier_file_path=PATH_TO_RESTRICTIONS_FILE)
+    tier = test_object.lacode_to_tier(lacode=lacode, date=datetime.now())
+    assert tier == 2
+
+
+def test_lacode_to_tier_invalid_lacode():
+    lacode = "invalid-la-code"
+    test_object = LacodeTierLookup(tier_file_path=PATH_TO_RESTRICTIONS_FILE)
+    tier = test_object.lacode_to_tier(lacode=lacode)
+    assert tier == DEFAULT_NATIONAL_ALERT_LEVEL
+
+
+@freeze_time('2020-10-21')
+def test_lacode_to_tier_single_restriction_mock_data():
+    test_object = LacodeTierLookup(tier_file_path=PATH_TO_RESTRICTIONS_FILE)
+    test_object.load_data = Mock()
+    test_object.load_data.return_value = fake_data()
+    tier = test_object.lacode_to_tier(lacode="lacode1", date=datetime.now())
+    assert tier == 1
+    tier = test_object.lacode_to_tier(lacode="lacode2", date=datetime.now())
+    assert tier == 2
+
+
+def test_lacode_to_tier_multiple_restriction_mock_data():
+    test_object = LacodeTierLookup(tier_file_path=PATH_TO_RESTRICTIONS_FILE)
+    test_object.load_data = Mock()
+    test_object.load_data.return_value = fake_data()
+    with freeze_time('2020-06-21'):
+        tier = test_object.lacode_to_tier(lacode="lacode3", date=datetime.now())
+        assert tier == DEFAULT_NATIONAL_ALERT_LEVEL
+    with freeze_time('2020-07-21'):
+        tier = test_object.lacode_to_tier(lacode="lacode3", date=datetime.now())
+        assert tier == 1
+    with freeze_time('2020-08-25'):
+        tier = test_object.lacode_to_tier(lacode="lacode3", date=datetime.now())
+        assert tier == 2
+    with freeze_time('2020-09-25'):
+        tier = test_object.lacode_to_tier(lacode="lacode3", date=datetime.now())
+        assert tier == 3
+    with freeze_time('2020-10-25'):
+        tier = test_object.lacode_to_tier(lacode="lacode3", date=datetime.now())
+        assert tier == 4
+    with freeze_time('2020-11-25'):
+        tier = test_object.lacode_to_tier(lacode="lacode3", date=datetime.now())
+        assert tier == 4
+
+
+def fake_data():
```

Comment:
We can probably worry about it later but it might be better to have a test `local-restrictions.yaml` so we can update the file that app uses in prod without possibly breaking tests.